docat/docat/nginx/default

```diff
@@ -23,5 +23,6 @@ server {
     }
 
     location / {
+        try_files $uri /index.html;
```

Comment:
ideally we support all of that but i am also very willing to tell people they need a `index.html` to make it work

Reply:
What this does is it redirects all requests to `/...` to the index.html file, which is the output of react build. This just makes sure that it falls back to the page, otherwise we would get an NGINX 404

Reply:
Ah, this is only for our internal ui, then we can leave it as is, sorry for the misunderstanding.